apps/web/playwright/manage-booking-questions.e2e.ts

```diff
@@ -262,6 +262,91 @@ test.describe("Manage Booking Questions", () => {
   });
 });
 
+test("Disable Input on url prefill for booking and verify a few thing in b/w", async ({
+  page,
+  users,
+  context,
+}, testInfo) => {
+  // Considering there are many steps in it, it would need more than default test timeout
+  test.setTimeout(testInfo.timeout * 2);
+  const user = await createAndLoginUserWithEventTypes({ users, page });
```

Comment:
I don't think we need `e2e` for this change. e2es are costly time wise and easily become flaky.

So, we are intentionally avoiding e2e wherever possible and focus on Unit tests.


This change is pretty straightforward. I would much prefer to have Unit tests for `FormBuilderField` and `FormBuilder` components instead.

Can be a followup PR as well.

Reply:
@hariombalhara  can you suggest any reference to work with this?

Reply:
You an look at some of *.test.tsx files that have react-testing-library based tests for components
![image](https://github.com/user-attachments/assets/e7a97927-a233-4464-8063-add64f19753f)

Reply:
> You an look at some of *.test.tsx files that have react-testing-library based tests for components ![image](https://private-user-images.githubusercontent.com/1780212/350347326-e7a97927-a233-4464-8063-add64f19753f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjEzODI4NzYsIm5iZiI6MTcyMTM4MjU3NiwicGF0aCI6Ii8xNzgwMjEyLzM1MDM0NzMyNi1lN2E5NzkyNy1hMjMzLTQ0NjQtODA2My1hZGQ2NGYxOTc1M2YucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDcxOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA3MTlUMDk0OTM2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZTUzMTM5NzdlZjI4YzE3ZmNlOWM3NWVmMjg0ZDFhODYzYTI4M2RkNjRlMGY1MDNhNTRkYjYwMDQ1ZDhkMGE2YyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.n0rCN78umy8GMgnHC5xjyc6Vv1KknYrpTD4TwWmSS1k)

thanks a lot. will check and update the pr